modular_skyrat/master_files/code/datums/id_trim/jobs.dm

```diff
@@ -64,8 +65,8 @@
 /datum/id_trim/job/nanotrasen_consultant
 	assignment = "Nanotrasen Consultant"
 	trim_state = "trim_centcom"
-	department_color = COLOR_GREEN
-	subdepartment_color = COLOR_GREEN
+	department_color = COLOR_COMMAND_BLUE // BUBBER EDIT - OLD: department_color = COLOR_GREEN
+	subdepartment_color = COLOR_CENTCOM_BLUE // BUBBER EDIT - OLD: dsubdepartment_color = COLOR_GREEN
```

Comment:
Especially not since you even do a base edit here! At that point just make a new define with reasonable wording!

Reply:
Actually this can just be modularized

Reply:
> Actually this can just be modularized

>more modularity pending soon

![Capture](https://github.com/Bubberstation/Bubberstation/assets/43571008/78f1eb55-8a13-4ffe-9b94-5a6086e36ee8)